facdb_build/sql/dcp_colp.sql

```diff
@@ -46,301 +47,301 @@ SET hash =  md5(CAST((t.*)AS text)),
 				ELSE geo_bbl
 			END),
 	facname = (CASE
-				WHEN (name = ' ' OR name IS NULL) AND usedec ~* 'office' THEN 'Offices'
-				WHEN (name = ' ' OR name IS NULL) AND usedec ~* 'no use' THEN 'City Owned Property'
-				WHEN name <> ' ' AND name IS NOT NULL THEN initcap(name)
-				ELSE initcap(REPLACE(usedec, 'OTHER ', ''))
+				WHEN (parcelname = ' ' OR parcelname IS NULL) AND usetype ~* 'office' THEN 'Offices'
+				WHEN (parcelname = ' ' OR parcelname IS NULL) AND usetype ~* 'no use' THEN 'City Owned Property'
```

Comment:
Should `AND ownership = 'C'` be added to this WHEN statement because a handful of records are not cityowned?